Test.C

```diff
@@ -178,8 +181,13 @@ int main(){
             while (b == 0){
                 // generates a random number between 0 and 6
                 randomAttackIndex = rand() % NUM_ATTACK;
+                if (existsInArray(randomAttackIndex, existingAttacks)) {
```

Comment:
No space after if statement before { per the style guide.
```suggestion
                if (existsInArray(randomAttackIndex, existingAttacks)){
```